pkg/core/blockchain.go

```diff
@@ -1340,6 +1340,10 @@ func (bc *Blockchain) verifyHeader(currHeader, prevHeader *block.Header) error {
 
 // verifyTx verifies whether a transaction is bonafide or not.
 func (bc *Blockchain) verifyTx(t *transaction.Transaction, block *block.Block) error {
+	height := bc.BlockHeight()
+	if t.ValidUntilBlock <= height || t.ValidUntilBlock > height+transaction.MaxValidUntilBlockIncrement {
+		return errors.Errorf("transaction has been expired. ValidUntilBlock = %d should be more than current blockchain height = %d and not exceed it more than %d", t.ValidUntilBlock, height, transaction.MaxValidUntilBlockIncrement)
```

Comment:
`has expired` and I think its enough to tell `ValidUntilBlock: %d, current height: %d`.